.ci/azure-pipelines.yml

```
@@ -87,15 +88,15 @@ jobs:
     displayName: 'Publish'
 
     dependsOn: Build
-    condition: startsWith(variables['Build.SourceBranch'], 'refs/tags')
+    condition: startsWith(variables['Build.SourceBranch'], 'refs/tags/v')
```

```suggestion
    condition: startsWith(variables['Build.SourceBranch'], 'refs/tags')
```
the `/v` should not be required unless you intent to ever have a tag that is not meant to trigger the `publish` job

---

This was a suggestion by @EraYaN so the publishing doesn't run for non-version tags (if we ever use them). Better be safe than sorry!

---

I see, but I did comment that too ;) (! suspected that be the reason behind this change, just wanted to be sure)